src/tests/test_nlvoorelkaar.py

```diff
@@ -1,11 +1,12 @@
 import os
+import unittest
```

Comment:
import alleen skip op de volgende regel ipv twee unittest imports

Reply:
@wkoot Zit nog maar één skip in en die import ik nu los.